### PR TITLE
new presence: lol espor

### PR DESCRIPTION
**Pull request by mertkaraca34**

### CODE REVIEW

**Comment by mertkaraca34** [comment]
image

File: websites/L/LoL Espor/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+  "$schema": "https://schemas.premid.app/metadata/1.0",
+  "author": {
+    "name": "mertkaraca.mp4",
+    "id": "749681979741569052"
+  },
+  "service": "LoL Espor",
```
Comment (Bas950) [inline websites/L/LoL Espor/dist/metadata.json:7]: This website doesnt seem to be called "LoL Espor" but "LoL Esports"

File: websites/L/LoL Espor/dist/metadata.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  "description": {
+    "tr":"League of legends oyununun Espor karşılaşmalarını canlı olarak izleyebileceğiniz platform.",
+    "en": "The platform where you can watch the Esports matches of the League of Legends game live."
+  },
+  "url": "lolesports.com",
+  "version": "1.0.0",
+  "logo": "https://i.imgur.com/CFlhiDo.jpg",
+  "thumbnail": "https://i.imgur.com/tmhf2Cm.jpg",
+  "color": "#3b1b8a",
+  "tags": ["lol", "leagueoflegends","espor","turkish","development","coding"],
```
Comment (Bas950) [inline websites/L/LoL Espor/dist/metadata.json:17]: Why is the turkish tag in here? The website seems from english origin

File: websites/L/LoL Espor/presence.ts [hunk 3]
```diff
@@ -0,0 +1,55 @@
+const presence = new Presence({
+  clientId: "759897044323794985"
+});
+
+const time = Math.floor(Date.now() / 1000);
```
Comment (Bas950) [inline websites/L/LoL Espor/presence.ts:5]: merge consts.

**Review by Bas950 — CHANGES_REQUESTED** [review]
Make the statuses in english

**Pull request closed without merging**